eng/helix/helix.proj

```diff
@@ -61,6 +76,40 @@
     <HelixProperties Condition="'$(RunQuarantinedTests)' != 'true'" Include="runType" Value="unquarantined" />
   </ItemGroup>
 
+  <Target Name="IncludeAspNetRuntime" BeforeTargets="Gather">
+    <MSBuild Projects="$(RepoRoot)src\Framework\App.Runtime\src\Microsoft.AspNetCore.App.Runtime.csproj"
+        Properties="DisableYarnCheck=true;ExcludeFromBuild=false"
+        Targets="_GetPackageVersionInfo"
+        SkipNonexistentProjects="false">
+      <Output TaskParameter="TargetOutputs" ItemName="_ResolvedProductVersionInfo" />
+    </MSBuild>
+
+    <PropertyGroup>
+      <SharedFxVersion>@(_ResolvedProductVersionInfo->'%(PackageVersion)')</SharedFxVersion>
+    </PropertyGroup>    
+    
+    <Unzip Condition="Exists('$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\Microsoft.AspNetCore.App.Runtime.$(TargetOsName)-$(TargetArchitecture).$(SharedFxVersion).nupkg')"
+       SourceFiles="$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\Microsoft.AspNetCore.App.Runtime.$(TargetOsName)-$(TargetArchitecture).$(SharedFxVersion).nupkg" 
+       DestinationFolder="$(OutputPath)\AspNetCoreAppRuntimeHelixRaw" />
+    <ItemGroup>
+      <_appRuntimeFiles Include="$(OutputPath)\AspNetCoreAppRuntimeHelixRaw\**\*.txt" />
+      <_appRuntimeFiles Include="$(OutputPath)\AspNetCoreAppRuntimeHelixRaw\**\*.json" />
+      <_appRuntimeFiles Include="$(OutputPath)\AspNetCoreAppRuntimeHelixRaw\**\*.dll" />
+      <_appRuntimeFiles Include="$(OutputPath)\AspNetCoreAppRuntimeHelixRaw\**\*.xml" />
+    </ItemGroup>
+    <Copy SourceFiles="@(_appRuntimeFiles)" DestinationFolder="$(OutputPath)\AspNetCoreAppRuntimeHelix\shared\Microsoft.AspNetCore.App\$(SharedFxVersion)" />
+    <Unzip Condition="Exists('$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\Microsoft.AspNetCore.App.Ref.$(SharedFxVersion).nupkg')" 
+      SourceFiles="$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\Microsoft.AspNetCore.App.Ref.$(SharedFxVersion).nupkg" 
+      DestinationFolder="$(OutputPath)\AspNetCoreAppRuntimeHelix\packs\Microsoft.AspNetCore.App.Ref\$(SharedFxVersion)" />
```

Comment:
nit: Without the include list approach you took with the Runtime, this will include the .nuspec file, _rels folder, and package folder. Suggest doing a similar thing here but with `$(RecursiveDir)` in the `<Copy />` task to handle the multiple folders you _do_ want in the Ref case.

Reply:
Does it hurt to include those files?  This mimics what we were doing before anyways, we used to just unzip everything into the folder in the test runner so I'd rather not tweak things any if its harmless for verification purposes.  My preference would be to just use unzip for both of these, but the runtime tests are currently written to expect a flattened hierarchy where the RuntimeList.xml isn't in the package folder.  Since the main purpose of this PR is to just get us switched onto the correlation payload functionality, I don't want to mess around with the app unit tests anymore since everything is passing.  I can file an issue to clean up this area for tracking purposes if you feel this is important in the future, but I don't want to tweak the msbuild anymore in this PR